tf-psa-crypto/drivers/builtin/src/.gitignore

```diff
@@ -1,4 +1,3 @@
 ###START_GENERATED_FILES###
 /error.c
-/version_features.c
```

Comment:
Minor, the commit message is refering to the the version.c file while we are removing the _features.c.

No need to rework as its pretty obvious what is happening.